_tools/install-lookout-latest.sh

```diff
@@ -22,19 +22,14 @@ if [[ "${PIPESTATUS[0]}" -ne 0 || "${PIPESTATUS[1]}" -ne 0 || "${PIPESTATUS[2]}"
   exit 2
 fi
 
-if ! tar -xvzf lookout_sdk_*.tar.gz ; then
+if ! tar -xvzf lookout-sdk_*.tar.gz ; then
   echo "Unable to extract lookout release archive" >&2
   exit 2
 fi
 
-if ! mv lookout_sdk_*/lookout .. ; then
-  echo "Unable to move lookout binary file" >&2
+if ! mv lookout-sdk_*/lookout-sdk ../lookout_sdk ; then
```

Comment:
Why the rename from `-sdk` to `_sdk`?

Reply:
because it's `lookout-sdk` dir we are in, and then `mv` binary with the same name to `../lookout_sdk` fails trying to overwrite it

Reply:
added a comment with clarifications